allennlp/models/coreference_resolution/coref.py

```diff
@@ -127,6 +129,9 @@ def forward(self,  # type: ignore
         span_labels : ``torch.IntTensor``, optional (default = None)
             A tensor of shape (batch_size, num_spans), representing the cluster ids
             of each span, or -1 for those which do not appear in any clusters.
+        output_alternative_antecedents: ``Optional[ScatterableList]`` - if non-`None` and
+            containing the single value ``True``, the output dictionary will contain antecedent
```

Comment:
This should now mention that `output_alternative_antecedents` may have length greater than one.

Reply:
Fixed